recipes/r-redlistr/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = '1.0.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-redlistr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/redlistr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/redlistr/redlistr_{{ version }}.tar.gz
+  sha256: 9949f23e668e7b560590993d3e0aa6502fb96fed6a67bbcfe13ebf20abeb7200
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-plyr
+    - r-raster >=2.5_8
+    - r-rgeos
+    - r-sp >=1.2_4
+  run:
+    - r-base
+    - r-plyr
+    - r-raster >=2.5_8
+    - r-rgeos
+    - r-sp >=1.2_4
+
+test:
+  commands:
+    - $R -e "library('redlistr')"           # [not win]
+    - "\"%R%\" -e \"library('redlistr')\""  # [win]
+
+about:
+  home: https://github.com/red-list-ecosystem/redlistr
+  license: GPL-3
```

Comment:
Should be `GPL-3.0-or-later`. This is the SPDX interpretation of `License: GPL (>= 3)`.